src/components/Description/Description.vue

```diff
@@ -35,21 +35,27 @@
 			@keydown.esc="handleCancelEditing" />
 		<template v-if="!loading">
 			<template v-if="editing">
-				<button class="nc-button nc-button__main description__action"
+				<Button type="tertiary"
+					class="description__action"
 					:aria-label="t('spreed', 'Cancel editing description')"
 					@click="handleCancelEditing">
-					<Close decorative
-						title=""
-						:size="16" />
-				</button>
-				<button class="nc-button nc-button__main primary description__action"
+					<template #icon>
+						<Close decorative
+							title=""
+							:size="16" />
```

Comment:
Button seems to not be visible... not sure what's wrong

Reply:
![Bildschirmfoto vom 2022-03-14 12-01-59](https://user-images.githubusercontent.com/213943/158159471-036e940e-2faf-4ecd-81b0-288015535471.png)

Reply:
for some reason hasIcon is false and text true 🤔 

<img width="174" alt="Screenshot 2022-03-15 at 09 57 59" src="https://user-images.githubusercontent.com/26852655/158342070-642b17c8-b019-4a89-bd53-364b2b88b2af.png">

Reply:
Umm, let me check it.

Reply:
Maybe from the broken lib?

Reply:
Yes, I think so.